pytorch_lightning/profiler/pytorch.py

```diff
@@ -374,10 +379,6 @@ def _default_activities(self) -> List['ProfilerActivity']:
             activities.append(ProfilerActivity.CUDA)
         return activities
 
-    @property
-    def step_action_names(self) -> Set[str]:
```

Comment:
shall this be properly deprecated?

Reply:
@tchaton ^^

Reply:
No, that function has not been released yet